### ISSUE TITLE
Associate and show CI environment info with each test run when run from a tool such as Github Actions

### ISSUE THREAD

**Issue by kdhamric** [kubeshop/tracetest#1066]
As a developer, I want to be able to see what pull request trigger a test run, what the commit was, etc when view test run info.  Look at the GitHub - cucumber/ci-environment: Detect CI Environment from environment variables

**Comment by mathnogueira** [comment]
Hey @kdhamric, do you have the list of information that would be good to have in the test run?

* latest commit
* PR number?
* PR url?

**Comment by kdhamric** [comment]
@mathnogueira I do not. We were adding this as @schoren found he needed info when viewing the tests - he would be the best source to get what fields to show. Looking at https://github.com/cucumber/ci-environment, it seems it captures a json object that has a CiEnvironment structure. Can we save the entire object or do we need to pick which fields to store?

**Comment by schoren** [comment]
I agree. Let's save everything available and decide what to show later
